searx/static/themes/simple/package.json

```diff
@@ -20,8 +20,8 @@
     "leaflet": "^1.7.1"
   },
   "scripts": {
-    "build": "npm install && grunt",
-    "start": "grunt watch",
-    "test": "grunt"
+    "all": "npm install && grunt",
```

Comment:
For my understanding: is `all` the default _target_ when calling `npm install`?

Reply:
See https://docs.npmjs.com/cli/v7/commands/npm-run-script
There is no default command, but there are some shortcuts:
* `npm start` ==> `npm run start`
* same for `restart`, `stop`, `test`.

In package.json of this PR:
* `npm run all` : download dependencies, build the theme
* `npm run build`: build the theme
* `npm run watch`: same spirit than `make docs-live` but for this theme instead of the documentation : any change to the source files recompile the theme ( https://github.com/gruntjs/grunt-contrib-watch )